interview-questions.md

```diff
@@ -7,60 +7,60 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a function? Why would you use one?
 
-  Your answer:
+  Your answer: A function is like a math equation that has variables and that output a soulution 
 
-  Researched answer:
+  Researched answer: A function is a piece of code that runs and returns a value.
 
 
 
 2. In regards to functions, what is the difference between an argument and a parameter?
 
-  Your answer:
+  Your answer: a parameter is almost like a equation that you want to have sovled, and an agurment are the variables for the parameters
 
-  Researched answer:
+  Researched answer: a parameter is a way to pass data into your code, arguments are the raw values that you want your function to work on.
 
 
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
+  Your answer: An index is the how data is stored.  Value is the type of data in a giving index.  The value of red is 0 [red, green, blue] 
```

Comment:
Really close but we do want to get this right: an index is **"where"** data is stored, and the value of [0] is red-- the index of red is [0]. If we flip it we can cause confusion-- not the end of the world right now but we do want to get that one right.